codegen/src/getCli.ts

```diff
@@ -24,24 +24,19 @@ export const getCli = async ({ spinner }: { spinner: Ora }) => {
 `
   );
   spinner.start('Looking up latest Xata CLI...');
-  const fileUrl = await fetch('https://api.github.com/repos/xataio/cli/releases/latest')
-    .then((r) => r.json())
+  const fileUrl = await fetch('https://api.github.com/repos/xataio/cli/releases/skizzy')
```

Comment:
I guess you changed the URL to test the error scenario but forgot to put it back to the right URL.

```suggestion
  const fileUrl = await fetch('https://api.github.com/repos/xataio/cli/releases/latest')
```